docs/oscar_references.bib

```diff
@@ -1019,6 +1026,18 @@ @Book{Stu93
   url           = {https://doi.org/10.1007/978-3-7091-4368-1}
 }
 
+@Article{Sym11,
+  author        = {Symonds, Peter},
+  title         = {On the {C}astelnuovo-{M}umford regularity of rings of
+                  polynomial invariants},
+  journal       = {Ann. of Math. (2)},
+  fjournal      = {Annals of Mathematics. Second Series},
+  volume        = {174},
+  year          = {2011},
+  number        = {1},
+  pages         = {499--517}
```

Comment:
Please add the DOI `10.4007/annals.2011.174.1.14`

(Bib items with DOI automatically get a hyperlink to the paper in the Oscar manual)